pkg/operator/context/apis.go

```diff
@@ -59,9 +85,37 @@ func getAPIs(config *userconfig.Config,
 					ResourceType: resource.APIType,
 				},
 			},
-			API:  apiConfig,
-			Path: context.APIPath(apiConfig.Name, config.App.Name),
+			API:                   apiConfig,
+			Path:                  context.APIPath(apiConfig.Name, config.App.Name),
+			RequestHandlerImplKey: requestHandlerImplKey,
+		}
+
+		if apiConfig.Requesthandler != nil {
+			uploadRequestHandlers(apis[apiConfig.Name], impls[*apiConfig.Requesthandler])
 		}
 	}
 	return apis, nil
 }
+
+func uploadRequestHandlers(api *context.API, impl []byte) error {
```

Comment:
Should it be called `uploadRequestHandler`? I can see it going either way, it's just for a sec I got confused and thought it was uploading multiple files